dockerfiles/jdk8/centos/is/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```

Comment:
Change the Licence Header

Reply:
fixed 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